### PR TITLE
tried some VC fixes.

### PR DESCRIPTION
**Pull request by scopatz**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,13 +16,15 @@ source:
...
 build:
   number: 0
-  skip: True  # [not py35]
   detect_binary_files_with_prefix: True
 
 requirements:
   build:
     - toolchain
     - cmake >=3.4
+    - vc 15  # [win]
```
Comment (isuruf) [inline recipe/meta.yaml]: This should be 14 (MSVC 14 - 2015)

**Comment by scopatz** [comment]
> Can you rerender the feedstock by running,

I am trying, but the latest conda-smithy seems totally broken on my system for whatever reason.

**Comment by scopatz** [comment]
```console
scopatz@artemis ~/re2-feedstock fixes $ conda smithy rerender
Traceback (most recent call last):
  File "/home/scopatz/miniconda3/bin/conda-smithy", line 4, in <module>
    import conda_smithy.cli
  File "/home/scopatz/miniconda3/lib/python3.5/site-packages/conda_smithy/cli.py", line 10, in <module>
    from conda_build.metadata import MetaData
  File "/home/scopatz/miniconda3/lib/python3.5/site-packages/conda_build/metadata.py", line 9, in <module>
    from .conda_interface import iteritems, PY3, text_type
  File "/home/scopatz/miniconda3/lib/python3.5/site-packages/conda_build/conda_interface.py", line 27, in <module>
    from conda.base.constants import SEARCH_PATH
ImportError: No module named 'conda.base'

```

**Comment by jakirkham** [comment]
Are you using the latest `conda-build-all`?

**Comment by jakirkham** [comment]
Also what versions of `conda-smithy`, `conda`, and `conda-build` are you using?

**Comment by scopatz** [comment]
Yeah I was using the latest versions of conda-smithy and conda-build-all (I did an update --all, right before that commad).  

The problem was conda istelf, which you have to manually update with the version number.  There is a probably a good reason for this....

**Comment by scopatz** [comment]
I was on conda v4.1.12, and the new conda-smithy requires conda v4.2.x. May it should be pinned as such?

**Comment by jakirkham** [comment]
Probably. Though really we should fix it to support that `conda` version. Can you please add an issue to `conda-smithy`'s issue tracker with this traceback?

**Comment by scopatz** [comment]
Sure,  I just opened up a PR, though too.

**Comment by jakirkham** [comment]
Great! 😀

**Comment by scopatz** [comment]
EIther of you know off hand how to set the VC compiler for CMake when using nmake?

**Comment by isuruf** [comment]
Can you try the following in meta.yaml?
```
build:
  msvc_compiler: 14.0
```

**Comment by scopatz** [comment]
Wow, thanks @isuruf! This seems to work now.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -16,13 +16,16 @@ source:
 
 build:
   number: 0
-  skip: True  # [not py35]
   detect_binary_files_with_prefix: True
+  msvc_compiler: 14.0  # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml]: So we still need the features I think. Try adding this...

```yaml
  features:
    - vc14  # [win]
```

**Comment by jakirkham** [comment]
Any thoughts on this, @msarahan ?

File: recipe/meta.yaml (hunk 2 above)
Comment (scopatz) [inline recipe/meta.yaml]: added!

**Pull request merged**